### PR TITLE
mcs: fix cannot build with swagger

### PR DESCRIPTION
**Pull request by nolouch**
### What problem does this PR solve?

Issue Number: Close https://github.com/tikv/pd/issues/6057

### What is changed and how does it work?

```commit-message
mcs: fix the resource manager cannot build with the swagger
```

### Check List

Tests

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
can build.

### Release note

```release-note
None.
```

### CODE REVIEW

**Review by lhy1024 — APPROVED** [review]

File: server/apiv2/handlers/keyspace.go [hunk 1]
```diff
@@ -171,10 +174,11 @@ type LoadAllKeyspacesResponse struct {
 }
 
 // LoadAllKeyspaces loads range of keyspaces.
+//
 // @Tags     keyspaces
 // @Summary  list keyspaces.
-// @Param    page_token  query  string  false  "page token"
-// @Param    limit       query  string  false  "maximum number of results to return"
+// @Param    page_token  query  string  false   "page token"
+// @Param    limit              query   string  false  "maximum number of results to return"
```
Comment (rleungx) [inline server/apiv2/handlers/keyspace.go:181]: The indentation looks strange.

File: server/apiv2/handlers/keyspace.go [hunk 2]
```diff
@@ -230,14 +234,16 @@ type UpdateConfigParams struct {
 // UpdateKeyspaceConfig updates target keyspace's config.
 // This api uses PATCH semantic and supports JSON Merge Patch.
 // format and processing rules.
+//
 // @Tags     keyspaces
 // @Summary  Update keyspace config.
-// @Param    name  path  string              true  "Keyspace Name"
+// @Param    name  path  string                        true  "Keyspace Name"
```
Comment (rleungx) [inline server/apiv2/handlers/keyspace.go:240]: ditto

Comment (nolouch) [reply server/apiv2/handlers/keyspace.go]: Updated, cc @rleungx  I format it with `swag fmt`.

**Review by rleungx — APPROVED** [review]

**Pull request merged**